filters/ubo-link-shorteners.txt

```diff
@@ -92,20 +92,20 @@ trip.businessnews-nigeria.com,te-it.com,world2our.com,mobi2c.com,bluetechno.net,
 trip.businessnews-nigeria.com,te-it.com,world2our.com,mobi2c.com,tech5s.co,ez4mods.com,bluetechno.net,forexit.online###submitBtn, #go_d, #devozon-snp, #submitBtn, #tp-snp1, #go_d2:style(display: block !important;)
 
 ! https://clk.asia/GIjT?src=PSA
```

Comment:
Could you add the working link here?

Reply:
@D4niloMR just to confirm, you want me to add `mythvista.com` to the 92nd line along with the other domains on that line, is that correct? This is my first time contributing to this repo so I am trying to adjust :)

Reply:
Thanks, but I'm asking you to replace the outdated link of `clk.asia` with the updated one in the place where `https://clk.asia/GIjT?src=PSA` is, since I have no way of testing without it.

Reply:
@D4niloMR I see, I reverted my previous commit & added a working `clk.asia` link for you to test. Thank you for your patience, this is my first time contributing to this repo, I am still learning.